src/components/pages/auth/login/index.tsx

```diff
@@ -11,9 +11,11 @@ import { toastify } from "@/helper/toastify";
 
 // API
 import { loginUser } from "@/backend/auth.api";
+import { account } from "@/backend/appwrite.config";
```

Comment:
configure the appwrite related stuff in the backend folder

Reply:
okay I will create a function in backend folder and import it here.

Reply:
Done, Moved it to backend folder